hazelcast/src/main/java/com/hazelcast/cluster/impl/ClusterClockImpl.java

```diff
@@ -46,4 +47,16 @@ public void setMasterTime(long masterTime) {
     public long getClusterTimeDiff() {
         return (clusterTimeDiff == Long.MAX_VALUE) ? 0 : clusterTimeDiff;
     }
+
+    @Override
+    public long getClusterStartTime() {
```

Comment:
Also what will happen if i set system clock back or forth one hour? How will this affect calculation? Should that cluster time be increased monotonically?